### PR TITLE
streamline settings

### PR DESCRIPTION
**Pull request by hotzen**

### CODE REVIEW

**Review by eratio08 — APPROVED** [review]

File: src/components/ModelChooser.tsx [hunk 1]
```diff
@@ -18,7 +18,6 @@ export function ModelChooser(props: {
         },
       })}
       data={[
-        { label: "GPT-4", value: "gpt-4" },
```
Comment (eratio08) [inline src/components/ModelChooser.tsx:21]: You also need to remove it from line 4

Comment (hotzen) [reply src/components/ModelChooser.tsx:21]: like that?

File: src/static/config.json [hunk 2]
```diff
@@ -1,160 +1,28 @@
...
-      "value": "A professional who helps individuals identify and achieve their personal and professional goals."
-    },
-    {
-      "label": "Career Counselor",
-      "value": "A professional who helps individuals explore and choose careers, develop job search strategies, and improve job performance."
-    },
-    {
-      "label": "Nutritionist",
-      "value": "A health professional who specializes in the study of nutrition and its effects on the body."
+      "label": "Software Engineer",
```
Comment (hotzen) [inline src/static/config.json:16]: just to streamline, not a proper prompt IMO

Comment (hotzen) [reply src/static/config.json:16]: could you provide a reasonable prompt that you actually use?

Comment (eratio08) [reply src/static/config.json:16]: I actually use the default softwareengineer + technical + concise ones and was always happy with the results so far. I need to dig into prompt engineering to give some proper input here

Comment (eratio08) [reply src/static/config.json]:
```
A professional who designs, develops, and maintains software applications and systems.
Answer as concise as possible
```
Writing Style
```
Technical
```
not sure if this is was you are asking for

Comment (hotzen) [reply src/static/config.json]: https://github.com/sportalliance/chatpad/pull/11

**Pull request merged**